### DISCUSSION TITLE
Exporting Documentation Outside of the App

### DISCUSSION THREAD

**Discussion by ericwang401** [dedoc/scramble#5] [Q&A] [answered]
Hello! I found your web app off of your post on Reddit here (https://www.reddit.com/r/laravel/comments/xguqz0/scramble_automatic_api_documentation_generator/) and I think it's absolutely neat!!

But I want to know if it's possible to host the documentation separate from the app? I am maintaining a project called Convoy and I want to host docs for people who use it so they can learn how to interface with the REST APIs (https://docs.convoypanel.com/guide/external-api/servers.html#fetching-servers).

I don't want to execute requests or tie the documentation in the source code of Convoy but display it using the beautiful design you've made in Scramble.

Please let me know if this is possible.

Thanks!

**Comment by romalytvynenko** [comment, marked as answer]
Hey @ericwang401!

Quick question before I can answer. 

So if I understood correctly, you already have OpenApi spec and simply want to show it in the design Scramble uses, but on other domain?

Or you need to generate docs with Scramble as well, and show it in the UI Scramble uses on other domain?

**Reply by ericwang401** [reply]
Hey thanks for replying!

I don't have Openapi specifications implemented in my code currently, and this documentation for the REST APIs in Convoy is all I have right now https://docs.convoypanel.com/guide/external-api/servers.html#fetching-servers and it looks really crude.

It would be awesome to generate the documentation (and host it on one domain) and provide that (but without the request sender tool) to everyone who deploys an instance of my app can refer to the API documentation. I might also need to manually add some APIs because I'm not sure if Scramble can parse the validation methods (I have some logic that gets the validation rules). 


So basically use the auto generating feature of Scramble (maybe some manual documenting as well) and use the awesome looking UI and host it on one domain for everyone.

I was curious if this was all possible.

**Reply by romalytvynenko** [reply]
Got it!

Yeah, sounds totally possible. Looks like first you'll need to install Scramble to your application (API) and generate spec. After you happy with the spec you can take it and host so it is available for your customers. 

I would be glad to help you with the setup. Here is a Discord server, feel free to join: https://discord.gg/Mk8AgsTRxD
